components/x-topic-search/src/AllFollowed.jsx

```diff
@@ -3,32 +3,32 @@ import styles from './TopicSearch.scss';
 import classNames from 'classnames';
 
 // transform like this => topic1, topic2 and topic3
-const arrayToSentence = matchingFollowedTopics => {
-	const topicsLength = matchingFollowedTopics.length;
+const arrayToSentence = followedSuggestions => {
+	const topicsLength = followedSuggestions.length;
 
 	if (topicsLength === 1) {
-		return <b>{ matchingFollowedTopics[0].name }</b>;
+		return <b>{ followedSuggestions[0].prefLabel }</b>;
 	} else {
-		return matchingFollowedTopics
+		return followedSuggestions
 		.map((topic, index) => {
 			if (index + 1 === topicsLength) {
```

Comment:
As per line 18, for some reason `if (index === topicsLength - 1)` is (to me) more readable. What do you think?

Reply:
I don't have a strong opinion with this. This part is come from the original I guess 🤔 But I think yours is more clean! I'm going to change it. Thank you!!